modules/ROOT/pages/index.adoc

```diff
@@ -38,6 +37,22 @@ endif::[]
 
 ====
 
+[WARNING]
+====
+Speech recognition ``mycroft-core`` and ``sphinx`` are available in PackageHub.
```

Comment:
```suggestion
Speech recognition ``mycroft-core`` and ``sphinx`` are available in Package Hub.
```
Not sure what's the right way to spell this but it would be good to use it consistently.